### DISCUSSION TITLE
Daily 20/9

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#16] [General]
Qué es lo que hicieron ayer? Qué es lo que planean hacer hoy? Ven alguna dificultad?

**Comment by rpgrca** [comment]
Ayer estuve agregando más información al ERS, hoy espero terminar la última parte que necesita información que todavía no tiene, además quiero terminar de ingresar a Github las historias de usuario más los issues para cada una, y al menos armar una nueva versión del ERS en formato PDF para subirla a la parte de documentación.

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve estudiando HTML, pensando US (agregue algunas US al documento)  y leyendo sobre IEEE830.
Hoy comenzaré con la lectura, en la plataforma, sobre CSS. También trataré de hacer la instalación del CMS.
Las dificultades pueden estar en alguna traba en la instalación del CMS y en la comprensíón de los nuevos contenidos.

**Comment by rocappellano** [comment]
Estuve con el tema de HTML, se me complica un poco uno de sus temas, sobre bits, más le estoy buscando la vuelta
Hoy seguiré estudiando sobre el tema, y buscaré ayuda en caso que se me complique aún más

**Comment by Brian-Almada** [comment]
Ayer estudié el material de SGdContenidos y estuve revisando el documento IEEE830 para conocer sus partes más en profundidad.
Hoy voy a analizar qué CMS puedo instalar para la tarea de SGdC y seguir con Python.

**Reply by AhumadaCeleste** [reply]
Ayer finalice de estudiar los apuntes de SGC. Avance con html. Pude generar enlaces en una prueba. Hoy continúe con dificultades en worpress. Cada vez que lo inicializo existen problemas con MySql. Pude resolverlo con mucho trabajo en eso. Mañana estudiare ingles principalmente y continuaré con html. 
No encuentro una buena definición de hipervínculos. Lo que encontré esta relacionado a documentos word y no se si, es un poco más amplio. Agradezco comentarios.

**Reply by rpgrca** [reply]
Un hipervínculo es un vínculo a alguna otra parte, ya sea otra parte del mismo documento, a otra parte de otro documento del mismo sitio o a un documento en un sitio completamente distinto.

**Comment by claubott** [comment]
Estoy poniendome al día con html y css.

**Comment by IngridAlcocer05** [comment]
Ayer continúe con la lectura de HTML y la guía de CMS. Hoy empecé con la lectura del modulo 4 de python2 y también instale XAMPP.

**Comment by Thiagox749** [comment]
Hoy resolvi un problema sobre XAMPP y sobre los puertos en MySql, además de clonar el repo.

**Comment by AhumadaCeleste** [comment]
GRACIAS!!!!!!!!!
